autosubmit/monitor/diagram.py

```diff
@@ -449,11 +452,12 @@ def _filter_by_status(jobs_list: List[Job]) -> List[Job]:
     return ret
 
 
-def _get_status(jobs_list: List[Job], job_name: str) -> str:
+def _get_status(jobs_list: List[Job], job_name: str) -> Optional[str]:
     """Return the status of the job."""
     for job in jobs_list:
         if job.name == job_name:
             return job.status_str
+    return None
```

Comment:
The rest are several IDE/linter errors, especially code comments, and functions that implicitly returned something (as in this line above), but doing explicitly is better for maintenance/readability.